src/servers/graphql-admin-server.ts

```diff
@@ -36,8 +36,9 @@ export async function startApolloServerForAdminSchema() {
     { allowExternalErrors: true },
   )
 
+  const port = process.env.PORT || 4001
```

Comment:
I think we might want to have different environment value, someone might want to have both the graphql-admin and graphql-main running, but with this PR you can do it only with the default settings

Reply:
probably the port name should be related to our software also?

Reply:
Fixed here https://github.com/GaloyMoney/galoy/pull/904/commits/292aeccaf515cefd092ceffcc5bab76a6e3d3162, removed v1 update because will be removed soon